specification/blueprint/resource-manager/Microsoft.Blueprint/preview/2017-11-11-preview/blueprint.json

```diff
@@ -0,0 +1,1707 @@
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/ManagementGroupsNamespaceParameter"
+          },
+          {
+            "$ref": "#/parameters/ManagementGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/BlueprintNameParameter"
+          },
+          {
+            "$ref": "#/parameters/PublishedBlueprintVersionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK -- published Blueprint retrieved.",
+            "schema": {
+              "$ref": "#/definitions/PublishedBlueprint"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "PublishedBlueprint"
+        ],
+        "operationId": "PublishedBlueprints_Delete",
+        "description": "Delete a published Blueprint.",
+        "x-ms-examples": {
+          "PublishedBlueprint_Delete": {
+            "$ref": "./examples/SealedBlueprint_Delete.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/ManagementGroupsNamespaceParameter"
+          },
+          {
+            "$ref": "#/parameters/ManagementGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/BlueprintNameParameter"
+          },
+          {
+            "$ref": "#/parameters/PublishedBlueprintVersionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK -- published Blueprint deleted.",
+            "schema": {
+              "$ref": "#/definitions/PublishedBlueprint"
+            }
+          },
+          "204": {
+            "description": "No Content"
+          }
+        }
+      }
+    },
+    "/providers/{managementGroupsNamespace}/managementGroups/{managementGroupName}/providers/Microsoft.Blueprint/Blueprints/{blueprintName}/versions": {
+      "get": {
+        "tags": [
+          "PublishedBlueprint"
+        ],
+        "operationId": "PublishedBlueprints_List",
+        "description": "List published versions of given Blueprint.",
+        "x-ms-examples": {
+          "PublishedBlueprintList": {
+            "$ref": "./examples/SealedBlueprint_List.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/ManagementGroupsNamespaceParameter"
+          },
+          {
+            "$ref": "#/parameters/ManagementGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/BlueprintNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK -- all published Blueprint retrieved.",
+            "schema": {
+              "$ref": "#/definitions/PublishedBlueprintList"
```

Comment:
Since publishedBluprint shares majority of the properties with bluprint itself, RBAC is a challenge here. Someone who was just assigned read on a specific version of bluprint, can still know all the properties of the parent, which is bluprint in this case.

Reply:
Something to keep in mind, and may be revisit the "version" story after preview.

Reply:
the version is a snapshot, the parent object is going to be modified and drift away.

Also we open Blueprint/read as unauthorizationed action, so everyone should be able to see the blueprint/artifact.